k480_conf.c

```diff
@@ -140,54 +140,66 @@ int main(int argc, char **argv)
 		}
 	}
 
-	/* Open the Device with non-blocking reads. */
-	fd = open(dev, O_RDWR|O_NONBLOCK);
-	if (fd < 0)
-	{
-		perror("Unable to open device");
-		return 1;
-	}
+	char devPath [30];
 
-	/* Get Raw Info */
-	res = ioctl(fd, HIDIOCGRAWINFO, &info);
-	if (res < 0)
-	{
-		perror("error while getting info from device");
-	}
-	else
-    {
-		if (info.bustype != BUS_BLUETOOTH ||
-		    info.vendor  != HID_VENDOR_ID_LOGITECH ||
-		   (info.product != HID_DEVICE_ID_K480 &&
-            info.product != HID_DEVICE_ID_K480_ALT &&
-            info.product != HID_DEVICE_ID_K480_ALT2))
+	/* TODO: get real device count? */
+	for (int i = 0; i < 10; i++) {
```

Comment:
@embuc I'm not familiar enough with the API - do you know the right way to get the real count of devices?

Reply:
No, not really, I remember that I briefly looked at it once but i was not able to figure it out quickly. It would be interesting to find out the proper way to do that though.